ui/mirage/scenarios/default.js

```diff
@@ -36,7 +36,40 @@ export default function(server) {
 
 function smallCluster(server) {
   server.createList('agent', 3);
-  server.createList('node', 5);
+
+  // server.createList('node', 5);
+  // FIXME data to exercise all permutations; remove before merging PR
```

Comment:
I know this isn't meant to stay around, but for future reference, [there is already a scenario for this](https://github.com/hashicorp/nomad/blob/master/ui/mirage/scenarios/default.js#L74-L85). Which gets set as `'allNodeTypes'` in [environment.js](https://github.com/hashicorp/nomad/blob/master/ui/config/environment.js#L25)

Reply:
Good to know, thanks!